### DISCUSSION TITLE
[FEATURE REQUEST] Custom Height Option

### DISCUSSION THREAD

**Discussion by xvim64** [upscayl/upscayl#827] [Ideas]
### Checklist

- [X] I have checked that this issue isn't addressed in [the FAQ](https://github.com/upscayl/upscayl#-faq).
- [X] I have searched for this in the [issues tab](https://github.com/upscayl/upscayl/issues?q=).

### Description

Sometimes the same height is very useful than the same width.

Is it possible to add a custom height option to the Upscayl?

That would be very nice.

**Comment by sfmarcius** [comment]
This would be extremely useful, especially in batch upscaling!

An alternative that I use is rotate all images beforehand with Multi Image Rotator (link below), do the upscale with the "custom output width" and unrotate everything after, but I dont know why, sometimes it messes with the borders when upscaling the rotated images (it renders a border with semitransparent pixels).

Multi Image Rotator - simple app to actually rotate images
https://www.softpedia.com/get/Multimedia/Graphic/Digital-Photo-Tools/Multi-Image-Rotator.shtml

**Comment by M4Mahdi** [comment]
I'm using the "rotation" workaround too. My Windows 10 has them(rotate to left and right) on right click of images. But why not implementing "Custom Output Height" option in the Upscayl?
![image](https://github.com/user-attachments/assets/9d4be67f-799a-4eb0-81c2-58da5e4c9041)
